README.md

```diff
@@ -46,28 +46,42 @@ Get help for a command:
 
     ./carddav2fb run -h
 
-#### Preconditions
+### Upload contact pictures
+
+Uploading can also be included in uploading phonebook:
+
+    ./carddav2fb run -i
+
+#### Settings
 
   * memory (USB stick) is indexed [Heimnetz -> Speicher (NAS) -> Speicher an der FRITZ!Box]
   * ftp access is active [Heimnetz -> Speicher (NAS) -> Heimnetzfreigabe]
+
+#### Preconditions
+
+  * requires FRITZ!Fon C4 or C5 handhelds
   * you use an standalone user (NOT! dslf-config) which has explicit permissions for FRITZ!Box settings, access to NAS content and read/write permission to all available memory [System -> FRITZ!Box-Benutzer -> [user] -> Berechtigungen]
 
 <img align="right" src="assets/fritzfon.png"/>
 
 ### Upload Fritz!FON background image
 
-Using the `background-image` command it is possible to upload the quickdial numbers as background image to FRITZ!Fon (nothing else!)
+The background image will be uploaded during
```

Comment:
Not happy with this change. What background image? Now the readme is missing every context what this command actually refers to.